lib/CL/pocl_llvm_wg.cc

```diff
@@ -23,11 +23,12 @@
    THE SOFTWARE.
 */
 
+#include "../llvmopencl/AutomaticLocals.h"
```

Comment:
Can you add llvmopencl to the include paths of this directory's CMake configuration instead?